classes/db/DAO.inc.php

```diff
@@ -136,6 +136,28 @@ function &retrieveCached($sql, $params = false, $secsToCache = 3600, $callHooks
 		return $result;
 	}
 
+	/**
+	 * Look if the current exception should be considered or not with SQL server.
+	 * @param $dataSource
+	 * @return bool
+	 */
+	function isSqlServerFakeError($dataSource) {
```

Comment:
@nibou230, I'm still wishing there was another way of dealing with this -- I think I already asked, but can't find it now: is this aspect of SQL server documented somewhere? (Sorry if I'm repeating myself.)

Reply:
I don't know why an error is raised at this point. It is related to how ADOdb is managing the queries or maybe how the system (we) use it. There is no exception when we run the generated queries from Laravel directly into the SQL manager.